chapters/introduction.txt

```diff
@@ -40,19 +39,15 @@ rendering-focused functionality only, so higher level scene graphs and other
 more general functionality must be obtained through other APIs or
 applications which use ANARI.
 
-Scientific visualization has diverse rendering needs involving tradeoffs
-among quality, speed, and scalability to available hardware resources. It is
-typical for visualization applications to use both visual and
-quantitative rendering techniques to satisfy user demands. Furthermore, it is
-common for an application to need rendering from local CPU or GPU hardware,
-with parallel scaling through multiple machines in a cluster to exploit
-additional distributed compute and memory resources. The ANARI API provides
-the necessary abstractions to allow these needs to be met by back-end
-renderers, without excessive exposure of hardware or rendering algorithm
-details to the application. ANARI seeks primarily to standardize emerging
-work found in scientific visualization, and also (where practical) to do the
-same for related domains, such as professional visualization, visual effects,
-and engineering CAD.
+Domains which leverage 3D graphics have diverse rendering needs involving
+tradeoffs among quality, speed, and scalability to available hardware resources.
+It is typical for 3D applications to use both visual and quantitative
+rendering techniques to satisfy user demands. Furthermore, it is common for an
+application to need rendering from local CPU or GPU hardware, with parallel
+scaling through multiple machines in a cluster to exploit additional distributed
+compute and memory resources. The ANARI API provides the necessary abstractions
+to allow these needs to be met by back-end renderers, without excessive exposure
+of hardware or rendering algorithm details to the application.
```

Comment:
Do we like to exemplarily enumerate some "targeted" domains? For example
> ANARI seeks to standardize features for rendering domains including, but not limited to, scientific visualization, professional visualization, visual effects, and engineering CAD.

Reply:
Addressed in 53193bc.